### PR TITLE
ft(S3C-4240): Allow reindexing a specific bucket

### PR DESCRIPTION
**Pull request by tmacro**
Adds a flag for manual use that allows the reindex to be limited to one bucket.

### CODE REVIEW

File: libV2/tasks/Reindex.js [hunk 1]
```diff
@@ -20,6 +20,16 @@ class ReindexTask extends BaseTask {
         this._defaultLag = 0;
     }
 
+    async _setup(includeDefaultOpts = true) {
+        await super._setup(includeDefaultOpts);
+        this._program.option(
+            '--bucket <bucket>',
+            'Manually specify a bucket to reindex. Can be specified multiple times.',
+            (v, p) => p.concat([v]),
```
Comment (naren-rajendran) [inline libV2/tasks/Reindex.js]: can you use a descriptive variable name, it will be easier to understand.

**Review by jonathan-gramain — APPROVED** [review]

**Review by naren-rajendran — APPROVED** [review]

**Pull request merged**